app/institutions/dashboard/-components/institutional-users-list/styles.scss

```diff
@@ -247,3 +252,38 @@ input:checked + .slider::before {
 .slider.round::before {
     border-radius: 50%;
 }
+
+.total-users {
+    margin-right: auto; /* Aligns text to the left */
+    display: flex; /* Allows for consistent alignment with other elements */
+    align-items: center;
+}
+
+.total-users label {
+    font-size: 18px;
+    margin-bottom: 0;
+    font-weight: normal;
+}
+
+.total-users-count {
+    font-size: 24px;
+    margin-right: 5px;
+    font-weight: bold;
+}
+
+/* Add media query for mobile devices */
+@media (max-width: 1000px) {
+    .table-wrapper {
+        overflow-x: auto;
+        -webkit-overflow-scrolling: touch; /* For smooth scrolling on iOS */
+    }
+
+    .table table {
+        min-width: 1000px; /* Set the table's width to be wider than the viewport */
```

Comment:
I worry about setting an explicit `min-width` here, as the table may not need the full 1000px as users can hide columns and whatnot... Can this be implemented without this hard-coded min-width?